daemon/remote/cmd/cmuxd-remote/agent_launch.go

```
@@ -146,6 +148,79 @@ func runOMORelay(socketPath string, args []string, refreshAddr func() string) in
+		return 1
+	}
+
+	// Native OMO Slim owns pane lifecycle; only inject socket/workspace context.
+	// Do not install plugins, create shadow config, or install a tmux shim.
+	launchContext, err := agentLaunchContextForInvocation(rc, omoLaunchIsNonLaunch(args))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "cmux %s: %v\n", commandName, err)
+		return 1
+	}
+
+	os.Setenv("CMUX_SOCKET_PATH", socketPath)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)